content/kubermatic/master/advanced/presets/_index.en.md

```diff
@@ -11,87 +11,117 @@ With Presets you can specify default settings for new Cluster. Use Presets to re
 
 ### Core concept
 
-As a Kubermatic administrator with superuser access, you can define Presets type in a standard format using a YAML
-definition, allowing the assignment of new credential types to supported providers. This allows you to define a custom
-credential type that works in ways similar to existing credential types. For example, you could create a custom credential
-type that injects access keys, passwords or network settings into Cloud object.
+As a Kubermatic administrator with superuser access, you can define Presets type in a standard format using a static YAML
+file or Kubernetes Custom Resource Definition (CRD) struct that represents the Preset, allowing the assignment of new
```

Comment:
Does it make sense to still document the presets.yaml here? The soon-to-be-used[hopefully] operator does not support them anymore.

Reply:
Yes, I agree but some customers still use the yaml file and current documentation is extremely outdated. Even yaml struct has changed so it's good to document this

Reply:
yes some customer will maybe use it longer as we expect